modules/mobile/app/helpers/mobile/appointments_cache_interface.rb

```diff
@@ -19,11 +19,11 @@ def fetch_appointments(user:, start_date: nil, end_date: nil, fetch_cache: true)
         end
       end
 
-      appointments = fetch_from_external_service(user, search_start_date, search_end_date)
+      appointments, failures = fetch_from_external_service(user, search_start_date, search_end_date)
       Mobile::V0::Appointment.set_cached(user, appointments)
 
       Rails.logger.info('mobile appointments service fetch', user_uuid: user.uuid)
-      appointments
+      [appointments, failures]
```

Comment:
What you're doing works, but I'm a little thrown off by the fact that this method returns both `appointments` on line 18 and `[appointments, failures]` here. I'm not sure if there's any general consensus on this, but I think the code would be clearer if we returned `[appointments, nil]` on line 18.

Reply:
fixed